lib/tool_shed/scripts/bootstrap_tool_shed/create_user_with_api_key.py

```diff
@@ -5,14 +5,15 @@
 import os
 import re
 import sys
-from optparse import OptionParser
+import argparse
```

Comment:
We are still using optparse in the rest of the codebase because argparse is not in Python 2.6, not sure if it's relevant for this script.